src/main/cpp/commands/drivetrain/TeleopDriveCommand.cpp

```diff
@@ -0,0 +1,26 @@
+
+#include "commands/drivetrain/TeleopDriveCommand.h"
+
+TeleopDriveCommand::TeleopDriveCommand(std::shared_ptr<Drivetrain> drivetrain,
```

Comment:
would we be able to make overloads of all the TeleOpDrive variants too? Just seems like there's a lot of repetitive code. Which is my fault for writing last year's code that way, of course, but still

Reply:
I agree. I'll create a design ticket for a next-gen command set for the drivetrain